core/modules/system/js/system.admin.themes.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file system.admin.themes.js
+ *
+ * Behaviors for themes admin.
+ */
+
+(function ($) {
+
+"use strict";
+
+/**
+ * Behavior for showing a list of disabled themes.
+ *
+ * Detect flexbox support for displaying our list of themes with vertical
```

Comment:
Wouldn't this entire thing be useful to be added globally instead of just the theme listing page? ...like the `.js` class being added to the `<html>` tag? That way, we'd be able to use it elsewhere, and contrib would also be able to take advantage of it.

Reply:
...referring to this: https://github.com/backdrop/backdrop/blob/1.x/core/misc/backdrop.js#L715